xml/bk_openstack_admin.xml

```diff
@@ -13263,9 +13260,7 @@ seeds the Python pseudo-random number generator.</para>
           <step>
             <para>The ring builder assigns each partition replica to the device that
 requires most partitions at that point while keeping it as far away
-as possible from other replicas. The ring builder prefers to assign a
-replica to a device in a region that does not already have a replica.
```

Comment:
Why are we removing references to the swift regions? this is true, and a region in Swift isn't necessarily equivalent to regions in keystone or nova.

Reply:
Thank you for the comments. Would you please suggest more accurate content?